plugins/woocommerce-blocks/assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -229,10 +285,11 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 						</span>
 					) }
 					<QuantityBadge
-						count={ productCount }
+						count={ cartItemsCount }
```

Comment:
instead of adding these props to the `QuantityBadge` could we determine whether the badge should be displayed at this level and conditionally render it?

Reply:
Since the `QuantityBadge` component includes both the cart icon and the product count badge.
So we can create a boolean prop like `showBadge` in the parent component and pass it down to it to determine whether the span with the class `wc-block-mini-cart__badge` should be visible. I'm not sure if this is the right approach. If you have any other suggestions, please let me know.